C/CUDA/CUTENSOR/build_tarballs.jl

```diff
@@ -8,11 +8,10 @@ include(joinpath(YGGDRASIL_DIR, "fancy_toys.jl"))
 include(joinpath(YGGDRASIL_DIR, "platforms", "cuda.jl"))
 
 name = "CUTENSOR"
-version = v"2.2.0"
-
+version = v"2.3.0"
 
 scripts = Dict()
-scripts[v"11"] = raw"""
+scripts[v"12"] = raw"""
```

Comment:
These scripts look very similar (or basically identical) with the exception of the version number. It would probably be better (and more maintainable) to just collapse them into one and have the version number as a interpolated string into it.